test/unit/org/apache/cassandra/schema/MockSchema.java

```diff
@@ -114,7 +116,7 @@ public static SSTableReader sstable(int generation, int size, boolean keepRef, l
         Descriptor descriptor = new Descriptor(cfs.getDirectories().getDirectoryForNewSSTables(),
                                                cfs.keyspace.getName(),
                                                cfs.getTableName(),
-                                               generation, SSTableFormat.Type.BIG);
+                                               new SequenceBasedSSTableUniqueIdentifier(generation), SSTableFormat.Type.BIG);
```

Comment:
I've demonstrated how we can accomplish testing with ULID and sequence identifier in `DirectoriesTest`. 

`MockSchema` is used is many places, effort to refactor all of those tests is a bit unpredictable. I'll try, but if it start appearing as it may take more than half a day, I'll have to extract it into a separate ticket - does this approach work for you?

Reply:
Ok. Yes, my intention was to suggest a separate ticket so as not to delay what's needed now for cndb.